cmd/serviceaccnt.go

```diff
@@ -0,0 +1,66 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	serivceAccount "github.com/method-security/methodk8s/internal/serviceaccount"
+	"github.com/spf13/cobra"
+)
+
+func (a *MethodK8s) InitServiceAccountCommand() {
+	serviceAccountCmd := &cobra.Command{
+
+		Use:   "service-account",
+		Short: "Service Account setup and config",
+		Long:  `Service Account setup and config`,
+	}
+
+	var namespace string
+	var apply bool
+	credentialsCmd := &cobra.Command{
+		Use:   "creds",
+		Short: "Service account credentials",
+		Long:  `Use this command to print to console the service account credentials`,
+		Run: func(cmd *cobra.Command, args []string) {
+			ctx := context.Background()
+			err := serivceAccount.PrintCredentials(ctx, a.K8Config, namespace)
+			if err != nil {
+				errorMessage := err.Error()
+				a.OutputSignal.ErrorMessage = &errorMessage
+				a.OutputSignal.Status = 1
+			}
+			a.OutputSignal.Content = nil
+		},
+		PersistentPostRunE: func(cmd *cobra.Command, _ []string) error {
+			return nil
+		},
+	}
+	configureCmd := &cobra.Command{
+		Use:   "config",
+		Short: "Set up service account in k8s cluster",
+		Long:  `Set up service account in k8s cluster`,
+		Run: func(cmd *cobra.Command, args []string) {
+			ctx := context.Background()
+			err := serivceAccount.Config(ctx, a.K8Config, apply, namespace)
+			if err != nil {
+				fmt.Println(err)
```

Comment:
Let's not use Println to print these things. We either want to use svc1log'ing (I can show you an example) or rely on the error messages in the signal output.